src/pages/team/index.tsx

```diff
@@ -180,13 +181,12 @@ export const getServerSideProps: GetServerSideProps<TeamPageProps> = async funct
     context.req.headers.cookie as string,
   )
 
-  const {data: teams} = await loadTeams(eggheadToken)
+  const {data: teams = []} = await loadTeams(eggheadToken)
 
-  let team: TeamData | undefined = undefined
+  const fetchedTeam = find(teams, (team) => team.capacity > 0)
```

Comment:
instead of assuming `0` (first) item is team account we will find an account with a capacity over `0` if it exists.

Also removed the `undefined` typing by adding an array as default.